packages/amplify-cli/src/domain/amplify-usageData/UsageDataPayload.ts

```diff
@@ -18,6 +18,7 @@ export class UsageDataPayload {
   isCi: boolean;
   accountId: string;
   projectSetting: ProjectSettings;
+  record: Record<string, any>[];
```

Comment:
Is this a record from accountID to log data ? Could you pl make it explicit with comments